src/tools/annotation/bidirectionalTool/utils/updatePerpendicularLineHandles.js

```diff
@@ -1,12 +1,27 @@
-// Update the  perpendicular line handles
-export default function(eventData, data) {
-  if (!data.handles.perpendicularStart.locked) {
-    return;
+import getLineVector from './getLineVector';
+
+/**
+ * Update the perpendicular line handles when the measurement is being created.
+ * This method will make the perpendicular line intersect in the middle of the
+ * long line and assume half the size of the long line.
+ *
+ * @param {*} eventData Data object associated with the event
+ * @param {*} measurementData Data from current bidirectional tool measurement
+ *
+ * @returns {boolean} False in case the handle is not locked or true when moved
+ */
+export default function updatePerpendicularLineHandles(
```

Comment:
`getUpdatedPerpendicularLineHandles`?

Reply:
The same applies here.
I don't want to touch upper levels of the tool code structure.